packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -23,14 +26,24 @@ export default class RatingBar extends PureComponent {
     score: 0,
     maxScore: 5,
     RatingItem: Star,
+    fillColor: 'black',
+  }
+
+  constructor(props) {
+    super(props)
+    this.effectiveId = this.props.id || randomId()
+  }
+
+  get id() {
+    return this.effectiveId
   }
 
   get ratingItems() {
     const {
+      id,
       maxScore,
       RatingItem,
       theme,
-      color,
```

Comment:
`fillColor` needs to be added here and called accordingly below (instead of `this.props.fillColor`), right now `fillColor` is being added as an invalid attribute on `RatingItem`.